1-Authentication/0-sign-in-vanillajs/App/public/redirect.html

```diff
@@ -3,4 +3,11 @@
     we recommend setting the redirectUri to a blank page or a page that does not implement MSAL.
     For more information, please follow this link: 
     https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/login-user.md#redirecturi-considerations 
--->
\ No newline at end of file
+-->
+<script src="/msal-browser.min.js"></script>
```

Comment:
Without the script tags, the redirect page won't actually redirect.

Reply:
The original intention for this page was for it to be used in the popup flow, where MSAL polls the URL from the main window. I think it'd be worth trying to just use /redirect as the redirectUri in the popup request and leave the redirect as localhost:3000 in the redirect request. This way you can remove these scripts from this page and leave it actually blank.